core/src/main/java/org/infinispan/container/entries/metadata/MetadataAware.java

```diff
@@ -3,7 +3,7 @@
 import org.infinispan.metadata.Metadata;
 
 /**
- * Metdata aware cache entry.
+ * Marker interface for metadata aware cache entry.
```

Comment:
@danberindei well, yeah, I planned on doing a cleanup (there's also an orphaned import somewhere there) while rebasing but forgot about it :)